zwave_js/DOCS.md

```diff
@@ -86,7 +86,7 @@ and your controller, before rebuilding your Z-Wave network.
 ### Option `log_level` (optional)
 
 This option sets the log level of Z-Wave JS. If no `log_level` is specified, the
-log level will be set to `info`. Valid options are:
+log level will be set to the level set in the Supervisor. Valid options are:
```

Comment:
This makes it sound like the valid options are for the Supervisor. Maybe move the sentence about the default log level after the list of valid options?